ReplayKitExample/README.md

```diff
@@ -67,20 +84,13 @@ The memory usage of a ReplayKit Broadcast Extension is limited to 50 MB (as of i
 
 It is highly recommended that you use Group Rooms with your ReplayKit extension, because the extension may connect without subscribing to Tracks.
 
-**2. Application Audio Delay**
-
-An `RPSampleHandler` may receive both application and microphone audio samples. We have found that, while microphone samples are suitable for realtime usage, application audio samples are significantly delayed.
+**2. Application Audio & Copy Protection**
 
-The following table shows what you can expect in the field (mesaured on iOS 12.1.3, and an iPhone X).
-
-| Sample Type | Format                          | Sample Size (Frames) | Period (milliseconds) |
-|-------------|---------------------------------|----------------------|-----------------------|
-| Application | 1ch, 44,100 Hz, Big Endian    | 22,596               | 512.3                 |
-| Microphone  | 1ch, 44,100 Hz, Little Endian | 1,024                | 23.2                  |
+It is not possible to capture application audio produced by AVPlayer, by Safari video playback (even if no Fairplay DRM is used), or by the Music app.
 
-**3. Application Audio & Copy Protection**
+**3. RPSystemBroadcastPickerView crashes (13.0-beta8)**
```

Comment:
Do you think worth adding a screen for this? Also it would be nice adding a developer forum link if any.

Reply:
Added a screenshot, linked to SO.